app/views/support_interface/data_exports/directory.html.erb

```diff
@@ -0,0 +1,28 @@
+<%= render 'support_interface/performance/performance_navigation', title: 'Data directory' %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <% @export_types.each do |id, export| %>
+
+      <% if @export_types.first.second[:name] != export[:name] %>
+        <section class="app-section app-section--with-top-border">
+      <% end %>
+        <h3 class="govuk-heading-m govuk-!-margin-bottom-2"><%= govuk_link_to export.fetch(:name), support_interface_view_export_information_path(data_export_type: id) %></h3>
+        <p class="govuk-body govuk-!-margin-bottom-2"><%= export.fetch(:description) %></p>
+      </section>
```

Comment:
@JR-G Each item should still be wrapped in a section, but the first section shouldn’t have the `app-section--with-top-border` class.